stripe/build.gradle

```diff
@@ -29,6 +29,8 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$androidLifecycleVersion"
     implementation "androidx.lifecycle:lifecycle-viewmodel-savedstate:$androidLifecycleVersion"
     implementation 'androidx.localbroadcastmanager:localbroadcastmanager:1.0.0'
+    implementation "androidx.fragment:fragment-ktx:1.3.0-alpha08"
```

Comment:
Discussed offline. Going to hold off on using ActivityResultContract until this is stable.